.travis.yml

```diff
@@ -10,7 +10,6 @@ matrix:
         - ARCHFLAGS="-std=c++11"
         - PIP_UPDATE="1"
         - PYTHON_BUILD=true
-        - BREW_BUNDLE=false
     
     - name: "klayout python2 osx10.12"
```

Comment:
do you also want to address the `brew` parts of this one in `MATRIX_EVAL`? I think it would be
```
addons:
  homebrew:
    packages:
      - python2
```

Reply:
No. I believe `brew install python2` triggers an error related to linking the executables, hence the second command. I don't think homebrew's add-on would work here. Thanks!